database/seed.ts

```diff
@@ -0,0 +1,185 @@
+import {Client} from "pg"
+import {database} from "./index";
+import {company, condominium, employee, department, scheduling, user, userType} from "./schema"
+import {fakerPT_BR as faker} from "@faker-js/faker";
+
+import * as dotenv from "dotenv";
+dotenv.config({ path: "./.env" });
```

Comment:
This can be omitted, since we're already injecting the environment variable trough the process spawn with the `package.json` script.

```suggestion
```